### PR TITLE
add changes for waifwu's new trailer

### PR DESCRIPTION
**Pull request by Avuxo**

### CODE REVIEW

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5.txt [hunk 1]
```diff
@@ -164,7 +164,7 @@ Leaving me behind, Arcueid leaps off from the rooftop and into the void.
 She's like a white flower, dancing in the wind.
 <Page47>
 An air current catches her body as she throws herself from the rooftop.
-The chains of gravity are broken,
+The chains of gravity release
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5.txt:167]: Comma drop

**Review by rschlaikjer — APPROVED** [review]

File: script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_5B.txt [hunk 2]
```diff
@@ -188,7 +188,7 @@ Don't ever look at things thoughtlessly, okay?"
 @gThere's no need to explain. It could only be one sound.
 @gThe sound of the knife being released.
 <Page40>
-@gAt that moment, I pass through the lines running along the woman's body.
+@gIn an instant, I trace the lines running along the woman's body.
 @gStab.
 @gCut.
 @gPass.
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_5B.txt:194]: Pass doesn't immediately read like a stabbing word. Jp is 通し, maybe replace with something like pierce? impale?

Comment (Avuxo) [reply script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_5B.txt:194]: Yeah I had a note about this in the channel not matching my suggestion for the verb form in 191. I'm happy with pierce.

File: script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_3_3.txt [hunk 3]
```diff
@@ -78,7 +78,7 @@ I'm served a teapot, a teacup, and some tea cakes.
 The snacks are miniature cakes and rice crackers sold at convenience stores for a certain price per bag, so it seems to be a place for just drinking tea and chatting.
 <Page24>
 "We're just here to chat, so let's not get too formal. That's no fun.
-You don't have to sit so formally or anything, you can relax and sit however you like, okay?"//正座とかしないで、足を崩しちゃってもいいですよ？」
+You don't have to be so stiff. You can relax and sit however you like, okay?"//正座とかしないで、足を崩しちゃってもいいですよ？」
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_3_3.txt:81]: repetition of "formal" reads a bit weird. 正座 localized as "stiff" and toka was dropped. Could be something like "and stuff" there but I think it flows better without it

File: script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_4_2.txt [hunk 4]
```diff
@@ -39,9 +39,9 @@ Kohaku-san nods.
 "That's very good. If you had, I would have needed to send you away."
 She replies with a smile while looking around the hallway.
 <Page13>
-"Fortunately, there's no one around.
-Now, please come in so we don't get spotted."
-She grabs my arm.
+"Fortunately, I don't see anybody around. //「幸いあたりに人影なし。
+Now, it'd be really bad if we got spotted out here so please come on in." // ささ、見つかるとタイヘンですからあがってくださいな」
+She snatches up my arm. //　ぐっ、と琥珀さんはこっちの腕を掴む。
```
[the remark this answers is not in the document]
Comment (yk1512) [reply script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_4_2.txt]: I prefer 'grabs hold of'

File: script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_3_3.txt (hunk 3 above)
Comment (rschlaikjer) [reply script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_3_3.txt:81]: This is _extremely_ minor, but three statements of fact like that in a row feels awkward for speech (at least to me). I might bring in an imperative sooner to break it up? Something like
> We're just here to chat, let's not get too formal. That's no fun.
> So stop sitting bolt upright like that, ok? Stretch your legs a little and relax.

File: script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_4_2.txt [hunk 5]
```diff
@@ -39,9 +39,9 @@ Kohaku-san nods.
 "That's very good. If you had, I would have needed to send you away."
 She replies with a smile while looking around the hallway.
 <Page13>
-"Fortunately, there's no one around.
-Now, please come in so we don't get spotted."
-She grabs my arm.
+"Fortunately, I don't see anybody around. //「幸いあたりに人影なし。
+Now, it'd be really bad if we got spotted out here so please come on in." // ささ、見つかるとタイヘンですからあがってくださいな」
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_4_2.txt:43]: I feel like 'really bad' might be a bit much. 
maybe smth like this?

> it'd be a problem if we got spotted out here

File: script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_2B.txt [hunk 6]
```diff
@@ -90,9 +90,9 @@ Human flesh and blood are also a source of nutrition for servants, but most of i
 The Castle Lord accumulates power as they sleep in their coffin, eventually turning the entire community into their own flesh and blood.
 ―――That is the nature of the vampire that lurks in this city."
 <Page28>
-"At the moment, they don't have all that many servants, so they're not that formidable. However, as the number of victims increases, so too does the power of a Castle Lord.
-I was hoping to find their coffin and kill them before that happens, but I still have yet to locate the enemy's resting place.
-This vampire has hidden themselves very skillfully, I can't sense their presence at all."
+"At the moment, they don't have all that many servants and so they're not that formidable. However, as the number of victims increases, so too does the power of a Castle Lord.
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_2B.txt:93]: **the** castle lord

File: script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_4_2.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_4_2.txt:44]: I'm not sure I agree that seizing is weird, but I think "grabs hold of" is better.

File: script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5.txt [hunk 7]
```diff
@@ -164,7 +164,7 @@ Leaving me behind, Arcueid leaps off from the rooftop and into the void.
 She's like a white flower, dancing in the wind.
 <Page47>
 An air current catches her body as she throws herself from the rooftop.
-The chains of gravity are broken,
+The chains of gravity release,
 the grounding anchors are raised,
 and the thing that should have fallen, rises gently instead.
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5.txt:169]: replace 'the thing' with 'what'?

File: script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_2B.txt (hunk 6 above)
Comment (rschlaikjer) [reply script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_2B.txt:93]: I think either might work here? It's generally true of castle lords, but also true of this specific one

Comment (Avuxo) [reply script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_2B.txt:93]: I read it as speaking in generalities since she specifies castle lord.

Comment (yk1512) [reply script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_2B.txt:93]: I wanted to keep consistent with line 90. Don't mind either way

**Review by yk1512 — APPROVED** [review]

**Pull request merged**